cloudbuild.yaml

```diff
@@ -28,15 +28,15 @@ steps:
   args:
   - 'build'
   - '-t'
-  - 'gcr.io/$PROJECT_ID/hello-cloudbuild:$SHORT_SHA'
+  - 'us-central1-docker.pkg.dev/$PROJECT_ID/my-repository/hello-cloudbuild:$SHORT_SHA'
   - '.'
 
-# This step pushes the image to Container Registry
+# This step pushes the image to Artifact Registry
 # The PROJECT_ID and SHORT_SHA variables are automatically
 # replaced by Cloud Build.
 - name: 'gcr.io/cloud-builders/docker'
```

Comment:
Name here should change as well to something unrelated to Container Registry. (Maybe 'artifacts/cloud-builders/docker" ?

Reply:
This is a nit BTW. If changing this you will need to change places that refer to the image name too

Reply:
I'm resolving this thread. 
See other thread for reason (Artifact Registry-hosted alternatives for the [official Cloud Build builder images](https://cloud.google.com/build/docs/cloud-builders#supported_builder_images_provided_by) are not yet ready).